spec/dummy/spec/packs_generator_spec.rb

```diff
@@ -234,6 +234,16 @@ module ReactOnRails
         end.not_to output(GENERATED_PACKS_CONSOLE_OUTPUT_REGEX).to_stdout
       end
 
+      it "adds a single import statement to the server bundle" do
+        test_string = "// import statement added by react_on_rails:generate_packs"
+        same_instance = described_class.instance
+        File.truncate(server_bundle_js_file_path, 0)
+        same_instance.generate_packs_if_stale
+        expect(File.read(server_bundle_js_file_path).scan(/(?=#{test_string})/).count).to equal(1)
```

Comment:
```suggestion
```
You can avoid double assertion here? It seems to test overall what you need.

Reply:
@alexeyr-ci I was thinking more in terms of the following:
1 - `generate_packs_if_stale` being called twice without no assertions in between.
2 - And then just one assertion at the end`expect(...).scan(/(?=#{test_string})/).count).to equal(1)`.

With the last `assertion` you validate two things:
- That the the link was generated properly.
- That even when you called it twice it only got created once(idempotent).

But also I understand your point that having that intermediate assertion in between validates that the url was created properly in the first call to `generate_packs_if_stale` and that the second assertion validates the idempotent behavior. That said I think it could be good to add that as a comment because even when you are using the same assertion and error/success in that assertion means something different.